kyo-os-lib/shared/src/main/scala/kyo/files.scala

```diff
@@ -1,139 +1,405 @@
 package kyo
 
+import java.io.IOException
+import java.nio.*
+import java.nio.channels.FileChannel
+import java.nio.charset.*
+import java.nio.file.*
+import java.nio.file.Path as JPath
+import java.nio.file.attribute.BasicFileAttributes
 import kyo.*
-import os.*
+import kyo.Flat.unsafe.*
```

Comment:
The flat check is crucial to ensure the correct execution of Kyo computations. I imagine you only need to pass the `Flat` implicit where it's required.